nuklear.h

```diff
@@ -16865,17 +16866,30 @@ nk_panel_end(struct nk_context *ctx)
         scaler.w = scrollbar_size.x;
         scaler.h = scrollbar_size.y;
         scaler.y = layout->bounds.y + layout->bounds.h;
-        scaler.x = layout->bounds.x + layout->bounds.w + panel_padding.x;
+
+        if (layout->flags & NK_WINDOW_SCALE_LEFT) {
+            scaler.x = layout->bounds.x - panel_padding.x / 2;
```

Comment:
@vurtun rebased this PR on to master but need you to verify this line. I have no idea if i did it correct here. Basically this is a guess that worked.